.github/workflows/pullrequest.yml

```diff
@@ -56,6 +78,21 @@ jobs:
         Get-Content "$env:temp\vcvars.txt" | Foreach-Object { if ($_ -match "^(.*?)=(.*)$") { Set-Content "env:\$($matches[1])" $matches[2] } }
         qmake -r MNECPP_CONFIG+=noTests
         jom -j2
+    - name: Configure and Compile BrainFlow package and submodule (Windows)
+      if: matrix.os == 'windows-2016'
+      run: |
+        # Setup VS compiler
+        cmd.exe /c "call `"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\vcvars64.bat`" && set > %temp%\vcvars.txt"
+        Get-Content "$env:temp\vcvars.txt" | Foreach-Object { if ($_ -match "^(.*?)=(.*)$") { Set-Content "env:\$($matches[1])" $matches[2] } }
+        cd applications\mne_scan\plugins\brainflowboard\brainflow
+        mkdir build
+        cd build
+        cmake -G "Visual Studio 14 2015 Win64" -DMSVC_RUNTIME=dynamic -DCMAKE_SYSTEM_VERSION=8.1 -DCMAKE_INSTALL_PREFIX=..\installed ..
+        cmake --build . --target install --config Release
+        cd ..\..
+        qmake -makefile brainflowboard.pro
+        jom -j2
+        cp -a .\brainflow\installed\lib\. ..\..\..\bin\
```

Comment:
Reproducing this on my local machine did not create the folder `/installed`, I think this is why windows is failing.

Reply:
I am hardly think so, here https://github.com/mne-tools/mne-cpp/pull/499/checks?check_run_id=467180946 it fails in the beginning while this cp is the last command

Reply:
or do you mean this "cmake -G "Visual Studio 14 2015 Win64" -DMSVC_RUNTIME=dynamic -DCMAKE_SYSTEM_VERSION=8.1 -DCMAKE_INSTALL_PREFIX=..\installed .." ?

Reply:
looks like I fixed it, dont need to run vcvars and I used wrong vs version in cmake generator, and cp command was wrong too